zeppelin-web/src/app/visualization/builtins/visualization-piechart.js

```diff
@@ -67,6 +65,9 @@ export default class PiechartVisualization extends Nvd3ChartVisualization {
   };
 
   configureChart(chart) {
-    chart.x(function(d) { return d.label;}).y(function(d) { return d.value;});
+    chart.x(function(d) { return d.label;})
+	 .y(function(d) { return d.value;})
+	 .showLabels(false)
+	 .showTooltipPercent(true);
```

Comment:
do you have a screen shot of before/after what it looks like? it would be handy for reviewers